### PR TITLE
Resolves #13 #19

### LINKED ISSUE #13: CI for dev,master testcases
Please add CI integration for run testcase when PR is made to dev, master branches.

We really need that!

### PR DESCRIPTION
**Pull request by Symone95**
Hi @GiovanniCardamone,
In 6c6201a027858e15df0cfa423b19d958dd7f4508 commit I've Implemented Behaviour class with inheritance to handle Ash's behaviours in various situations

Simone

### CODE REVIEW

**Comment by GiovanniCardamone** [comment]
Send me a PM when this is ready

**Comment by Symone95** [comment]
Hi @GiovanniCardamone,
I've implemented poetry to run testcases in this commit: 2af2cdf6b1e199843ec3a3749d7e3a07a73c3388

Simone

**Comment by GiovanniCardamone** [comment]
the test fails https://github.com/BeatingCodingame/python3-code-vs-zombie/pull/18/checks

**Comment by GiovanniCardamone** [comment]
i will do a review of this PR, please fix it so i can merge

File: .github/workflow/python-package.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
```
Comment (GiovanniCardamone) [inline .github/workflow/python-package.yml:28]: you should use only poetry to install the dependencies

File: .github/workflow/python-package.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      with:
+        python-version: 3.8
+    - uses: Gr1N/setup-poetry@v4
+    - uses: actions/cache@v1
+      with:
+        path: ~/.cache/pypoetry/virtualenvs
+        key: ${{ runner.os }}-poetry-${{ hashFiles('poetry.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-poetry-
+    - run: poetry run pytest
```
Comment (GiovanniCardamone) [inline .github/workflow/python-package.yml:48]: missing dependencies install using poetry

**Review by GiovanniCardamone — CHANGES_REQUESTED** [review]
The actions should only use poetry as dependencies manager

**Comment by GiovanniCardamone** [comment]
rejected

**Pull request closed without merging**